pastas/model.py

```diff
@@ -1553,25 +1557,21 @@ def get_block_response(
             Adds 0 at t=0 at the start of the response, defaults to False.
         dt: float, optional
             timestep for the response function.
-        kwargs: dict
+        kwargs: dict, optional
+            Kwargs are passed onto _get_response()
 
         Returns
         -------
         b: pandas.Series or None
             Pandas.Series with the block response. The index is based on the
             frequency that is present in the model.settings.
         """
-        return self._get_response(
-            block_or_step="block", name=name, dt=dt, p=p, add_0=add_0, **kwargs
-        )
+        return self._get_response(block_or_step="block", name=name, **kwargs)
 
     @get_stressmodel
     def get_step_response(
         self,
         name,
-        p: Optional[ArrayLike] = None,
```

Comment:
Why are these arguments removed? I don't think this is necessary and this should be retained. Same for get_block_response.

Reply:
There is no need to specifically name these arguments, as the kwargs are already passed onto _get_response, but I will change it back. May be easier for someone to see these parameters directly.

Reply:
I was specifically thinking about the case when people use this method and want to change the input, e.g., I use add0 every now and then so it's nice that these features are documented.

Reply:
They are still named in the docstring, so they are documented. However, I have just added them again as arguments as well.